internal/utils/helper.go

```diff
@@ -34,25 +35,32 @@ var (
 	locker         sync.Mutex
 )
 
-func MessageClientID() string {
+func init() {
+	buf := new(bytes.Buffer)
+
+	ip, err := clientIP4()
+	if err != nil {
+		ip = fakeIP()
+	}
+	buf.Write(ip)
+	binary.Write(buf, binary.BigEndian, Pid())
+	binary.Write(buf, binary.BigEndian, ClassLoaderID())
+	prefix = strings.ToUpper(hex.EncodeToString(buf.Bytes()))
+}
+
+func CreateUniqID() string {
```

Comment:
the func shouldn't in here, move to `trace.go`

Reply:
done